src/features/staking/subcomponents/ManageStake.tsx

```diff
@@ -95,7 +61,7 @@ const ManageStake: FC<ManageStakeProps> = ({
         <div className="flex flex-col text-right  uppercase">
           <div>
             <input
```

Comment:
Note: this `<input>` was refactored into a new component in PR: https://github.com/airswap/airswap-voter-rewards/pull/30